java/google/registry/rde/imports/RdeImportUtils.java

```diff
@@ -83,9 +101,15 @@ public RdeImportUtils(
         type,
         resource.getForeignKey(),
         resource.getRepoId());
-    ofy.save().entity(resource);
-    ofy.save().entity(ForeignKeyIndex.create(resource, resource.getDeletionTime()));
-    ofy.save().entity(EppResourceIndex.create(Key.create(resource)));
+    return ImmutableSet.<Object>of(ForeignKeyIndex.create(resource, resource.getDeletionTime()),
```

Comment:
This method is named "get" but it's really creating the two types of indexes.  Should the method instead be named "create"?